pkg/jobs/registry.go

```diff
@@ -741,7 +741,7 @@ func (r *Registry) maybeAdoptJob(ctx context.Context, nl NodeLiveness) error {
 			// Don't try to start any more jobs unless we're really live,
 			// otherwise we'd just immediately cancel them.
 			if liveness.NodeID == r.nodeID.Get() {
-				if !liveness.IsLive(r.clock.Now()) {
+				if !liveness.IsLive(r.clock.Now().GoTime()) {
```

Comment:
PhysicalNow

Reply:
actually, I switched everybody the other way. See note at the top.